assembler/manual.html

```diff
@@ -706,13 +708,13 @@ <h4>Input data</h4>
 
 <p>
     <code>--pacbio &lt;file_name> </code><br>
-    &nbsp;&nbsp;&nbsp;&nbsp;File with PacBio CLR reads. For PacBio CCS reads use <code>-s</code> option. More information on PacBio reads is provided in <a href="#pacbio">section 3.1</a>. 
+    &nbsp;&nbsp;&nbsp;&nbsp;File with PacBio CLR reads for late scaffolding step. For PacBio CCS reads use <code>-s</code> option. More information on PacBio reads is provided in <a href="#pacbio">section 3.1</a>.
 </p>
 
 
 <p>
     <code>--nanopore &lt;file_name> </code><br>
-    &nbsp;&nbsp;&nbsp;&nbsp;File with Oxford Nanopore reads. 
+    &nbsp;&nbsp;&nbsp;&nbsp;File with Oxford Nanopore reads for late scaffolding step.
```

Comment:
The change is not quite correct. There is no "late scaffolding" step in SPAdes at all.